app/components/task_list_items/review_immunity_detail_permitted_development_right_component.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module TaskListItems
+  class ReviewImmunityDetailPermittedDevelopmentRightComponent < TaskListItems::BaseComponent
+    def initialize(planning_application:)
+      @planning_application = planning_application
+    end
+
+    private
+
+    attr_reader :planning_application
+
+    delegate(:permitted_development_right, to: :planning_application)
+    delegate(:immunity_detail, to: :planning_application)
```

Comment:
Think you can do 
```
    delegate(
      :permitted_development_right,
      :immunity_detail,
      to: :planning_application
    )
```

Reply:
Don't actually need the `permitted_development_right` here so will remove that